docs/design/Profiles.md

```diff
@@ -0,0 +1,52 @@
+# Profiles
+
+## Overview
+
+Profiles is a configuration format and feature of Zonemaster Engine.
+Each of Zonemaster CLI, Backend and GUI allow users and administrators to configure Zonemaster Engine with different profiles.
```

Comment:
I think we should have a comment here that what was config and policy in previous versions of Engine and CLI have been integrated into profile. And that what was profile in previous version of Backend has now been expanded to be a full configuration of Zonemaster Engine.

Reply:
I can add that to the Background section as a historical note. Also mentioning that the profiles feature was added in 2017.4. Are you okay with that?

Reply:
Yes, that sounds good.

Reply:
I only rephrased the background section a bit. I opted not to add details about the relation between profiles and the various components. I'll add things to the component sections if there's anything more that needs to be highlighted.